### ISSUE TITLE
alvar comments - five year outlook data

### ISSUE THREAD

**Issue by alvizek** [FlowWest/urban-water-drought-data#6]
- [ ] Is there any way to obtain the supply data by supply source? Without the supply source it is difficult to know how realistic these numbers are.

- [ ] The population for each of the forecast periods could be an interesting data point (to assess demands using demand per capita)

**Comment by zoekanavas** [comment]
@alvizek do you think Alvar was referring to the supply source type (spring, well, consecutive connection, intake etc.) or water type (groundwater or surface water)? Or something else?

**Comment by zoekanavas** [comment]
we do not have the forecasted population, but we could include the current population.

**Comment by alvizek** [comment]
Yes I think he was probably referring to supply source type. I'd have to look to see if that is something included in the UWMP - do you know? Is population included in UWMP or we could pull in from another source?

**Comment by zoekanavas** [comment]
oh oh oh! we DO have forecasted population. This is included with the UWMP. The "current" population is field POPULATION_SERVED_2020. The forecasted populations are fields POPULATION_SERVED_2025, POPULATION_SERVED_2030, POPULATION_SERVED_2035, POPULATION_SERVED_2040, POPULATION_SERVED_2045. These fields are in 3-1_Retail and 3-1_Wholesaler. However, we don't have the projected population for the "between" years (i.e., 2021, 2022, 2023, 2024, etc.).

**Comment by zoekanavas** [comment]
we also have the source type included with the UWMP, this is called SOURCE and is in table SB_X7-7_CF_Table_4-A

**Comment by alvizek** [comment]
Zoe and Ashley decided that due to timeline we are not going to implement these changes now but want to return this. Instead we are going to add language to data dictionaries that pwsid and org_id can be used to link tables
